docs/source/release/v3.8.0/ui.rst

```diff
@@ -22,6 +22,12 @@ User Interface
 Instrument View
 ###############
 
+Workspace Matrix View
+#####################
+* Masked spectra will be highlighted with background colour (typically light grey).
+* Monitor spectra will have a dynamic background colour depending on the system settings (typically light yellow, as before).
+.. figure::  ../../images/MaskedAndMonitor.png
+
```

Comment:
It is not clear what colour a detector will be shown if it is both masked and a monitor. I'll find out what this is when I test again.